blog/content/post/2021-04-18-optiplex_part2.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Dell OptiPlex and coreboot - a story about porting cursed hardware (part 2)
+abstract: 'This post is a second part of the series which tells a story about
+porting Dell OptiPlex 9010 workstation to open-source firmware - coreboot. What
+was simple is long past us in the first part of the series. This time we are
+taking a deeper dive into the technical aspects of Environmental Controller
+reverse engineering and locating the problem with machine not booting after AC
+loss.'
```

Comment:
Not a part of this PR. We can just copy that to the spreadsheet and work on it there.

Reply:
@pietrushnic I don't see anything in the repo that would say anything about limiting abstract length. If you have more goals in mind, then maybe write them down please somewhere in README?

Reply:
@miczyg1 not obvious but:

> post abstract in the header is required for the posts summary in the blog list and must contain from 3 to 5 sentences, please note that abstract would be used for social media and because of that should be focused on keywords/hashtags

https://github.com/3mdeb/news-and-ideas/blob/master/blog/content/post/YYYY-MM-DD-template-post.md